pkg/gitreceive/k8s_util_test.go

```diff
@@ -53,18 +55,18 @@ func TestBuildPod(t *testing.T) {
 	var pod *api.Pod
 
 	slugBuilds := []slugBuildCase{
-		{true, true, "test", "default", emptyEnv, "tar", "put-url", ""},
-		{true, false, "test", "default", emptyEnv, "tar", "put-url", ""},
-		{true, true, "test", "default", env, "tar", "put-url", ""},
-		{true, false, "test", "default", env, "tar", "put-url", ""},
-		{true, true, "test", "default", emptyEnv, "tar", "put-url", "buildpack"},
-		{true, false, "test", "default", emptyEnv, "tar", "put-url", "buildpack"},
-		{true, true, "test", "default", env, "tar", "put-url", "buildpack"},
-		{true, false, "test", "default", env, "tar", "put-url", "buildpack"},
+		{true, true, "test", "default", emptyEnv, "tar", "put-url", "", ""},
+		{true, false, "test", "default", emptyEnv, "tar", "put-url", "", ""},
+		{true, true, "test", "default", env, "tar", "put-url", "", ""},
+		{true, false, "test", "default", env, "tar", "put-url", "", ""},
+		{true, true, "test", "default", emptyEnv, "tar", "put-url", "buildpack", ""},
+		{true, false, "test", "default", emptyEnv, "tar", "put-url", "buildpack", ""},
+		{true, true, "test", "default", env, "tar", "put-url", "buildpack", ""},
+		{true, false, "test", "default", env, "tar", "put-url", "buildpack", ""},
```

Comment:
could you add a row here to test for a custom image name? same with the below table for docker builds